ktor-server/ktor-server-plugins/ktor-server-di/common/src/io/ktor/server/plugins/di/DependencyProvider.kt

```diff
@@ -44,6 +44,14 @@ public interface DependencyProvider {
     public val declarations: Map<DependencyKey, DependencyCreateFunction>
 }
 
+/**
+ * Basic call for providing a dependency, like `provide<Service> { ServiceImpl() }`.
+ */
+public inline fun <reified T> DependencyProvider.provide(
+    name: String? = null,
+    noinline provide: DependencyResolver.() -> T
+): Unit = set(DependencyKey(typeInfo<T>(), name), provide)
```

Comment:
please avoid using `=` for functions with Unit return type

Reply:
@coderabbitai, remember this rule